xml/docu_styleguide.language.xml

```diff
@@ -452,7 +452,16 @@
    require more specific wording.
   </para>
   <para>
-   For instructions concerning markup, refer to
+   When referring to UI labels, capitalize them exactly as in the UI itself.
+   Software created at &suse; (such as YaST or Uyuni) should use
+   sentence-style capitalization.
+   If it does not, you can make aware the developers of that software.
```

Comment:
```suggestion
   As a general rule, &suse; (such as YaST or Uyuni) should use sentence-style
   capitalization in our software. If we do not, writers should make the developers
   aware of this guideline.
```,

Reply:
@lproven weren't you the one who came up with KISS? IHMO we should leave the sentence as is and not make it more complicated. Sentence-style is a company-wide standard. And the docs team does not develop software, so we don't care if sentence-case is just a general rule (with valid exceptions) or a strict SUSE brand standard.

Reply:
@lproven Besides being a bit more complicated, your suggestion seems broken too: "(YaST or Uyuni)" needs to go after "...software" and there is also an extra line at the end.

Using "we" here, for the entire company, makes this a bit too personal for me too^^. As a writer, I wouldn't feel chiefly responsible for actually fixing/creating software, so please don't we me in there.

Reply:
Ok, after 14 days with no further comments, I am just going to merge.